src/Microsoft.ML.CodeGenerator/Templates/Azure/Console/AzureModelBuilder.cs

```diff
@@ -0,0 +1,430 @@
+﻿// ------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version: 16.0.0.0
+//  
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+// ------------------------------------------------------------------------------
+namespace Microsoft.ML.CodeGenerator.Templates.Azure.Console
+{
+    using System.Linq;
+    using System.Text;
+    using System.Collections.Generic;
+    using System.Text.RegularExpressions;
+    using System;
+    
+    /// <summary>
+    /// Class to produce the template output
+    /// </summary>
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    internal partial class AzureModelBuilder : AzureModelBuilderBase
+    {
+        /// <summary>
+        /// Create the template output
+        /// </summary>
+        public virtual string TransformText()
+        {
+if(Target == CSharp.GenerateTarget.Cli){ 
+CLI_Annotation();
+ } else if(Target == CSharp.GenerateTarget.ModelBuilder){ 
+MB_Annotation();
+ } 
+            this.Write("using System;\r\nusing System.Collections.Generic;\r\nusing System.IO;\r\nusing System." +
+                    "Linq;\r\nusing Microsoft.ML;\r\nusing Microsoft.ML.Data;\r\nusing ");
+            this.Write(this.ToStringHelper.ToStringWithCulture(Namespace));
+            this.Write(".Model;\r\nnamespace ");
+            this.Write(this.ToStringHelper.ToStringWithCulture(Namespace));
+            this.Write(".ConsoleApp\r\n{\r\n    public static class ModelBuilder\r\n    {\r\n        private stat" +
+                    "ic string TRAIN_DATA_FILEPATH = @\"");
+            this.Write(this.ToStringHelper.ToStringWithCulture(Path));
+            this.Write("\";\r\n        private static string MLNET_MODEL = @\"");
+            this.Write(this.ToStringHelper.ToStringWithCulture(MLNetModelpath));
+            this.Write("\";\r\n\t\tprivate static string ONNX_MODEL = @\"");
+            this.Write(this.ToStringHelper.ToStringWithCulture(OnnxModelPath));
+            this.Write(@""";
+       
+	    // Create MLContext to be shared across the model creation workflow objects 
+        // Set a random seed for repeatable/deterministic results across multiple trainings.
+        private static MLContext mlContext = new MLContext(seed: 1);
+
+		// Training in Azure produces an ONNX model; this method demonstrates creating an ML.NET model (MLModel.zip) from the ONNX model (bestModel.onnx), which you can then use with the ConsumeModel() method to make predictions
```

Comment:
This line uses tabs instead of spaces.

Reply:
This is auto-generated file, I can also do nothing about it's format

Reply:
The lines above use spaces, a few of the lines here use tabs. We should have control of that in the file which produces this one.

Reply:
Oh do you mean the space/tab in comment and code intent from line 47 to line 52?

Reply:
## Resolved